README.md

```diff
@@ -37,6 +41,31 @@ b = Dagger.@spawn rand(a, 4)
 c = Dagger.@spawn sum(b)
 fetch(c) # some number!
 ```
+## Contributing Guide
+[![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat-square)](http://makeapullrequest.com)
+![GitHub issues](https://img.shields.io/github/issues/JuliaParallel/Dagger.jl)
+![GitHub contributors](https://img.shields.io/github/contributors/JuliaParallel/Dagger.jl)
+[![GitHub](https://badgen.net/badge/icon/github?icon=github&label)](https://github.com)
+
+Contributions are encouraged. 
+
+There are several ways to contribute to our project:
+
+**Reporting Bugs**: If you find a bug, please open an issue and describe the problem. Make sure to include steps to reproduce the issue and any error messages you receive regarding that issue.
+
+**Fixing Bugs**: If you'd like to fix a bug, please create a pull request with your changes. Make sure to include a description of the problem and how your changes will address it.
+
+Additional examples and documentation improvements are also very welcome.
+
+## Documentation
+List of recommended Dagger resources:
```

Comment:
Would you mind also listing the video links that I sent to you all in Slack? I think they're useful resources to point contributors to.

Reply:
yes, sure. I'll add them.